config/mapping.csv

```diff
@@ -379,7 +379,7 @@ ScreenEffect::screenEffectDeletedCallback,0x430220,0x4b,default,u32,ScreenEffect
 SoundPlayer::InitializeDSound,0x430270,0x29d,__thiscall,void,SoundPlayer*,HWND__*
 SoundPlayer::Release,0x430510,0x1cd,__thiscall,ZunResult,SoundPlayer*
 SoundPlayer::LoadSound,0x4306e0,0x310,__thiscall,ZunResult,SoundPlayer*,i32,char*
-SoundPlayer::GetWavFormatData,0x4309f0,0x57,default,WAVEFORMATEX*,i32*,char*,i32*,i32
```

Comment:
Don't make changes to the mapping.csv directly. Instead, make your changes in ghidra, and let the update mapping PR pull them in.